app/controllers/CommodityCodeController.scala

```diff
@@ -52,21 +52,37 @@ class CommodityCodeController @Inject() (
 
   private val form = formProvider()
 
-  def onPageLoad(mode: Mode): Action[AnyContent] = (identify andThen getData andThen requireData) { implicit request =>
-    val preparedForm = request.userAnswers.get(CommodityCodePage) match {
-      case None        => form
-      case Some(value) => form.fill(value)
-    }
+  def onPageLoadCreate(mode: Mode): Action[AnyContent] = (identify andThen getData andThen requireData) {
```

Comment:
looks good :), just this one needs scala formatting for CommodityCodeController.scala